### ISSUE TITLE
dist-git-source with gpg signature in sources fails to run

### ISSUE THREAD

**Issue by bcl** [teemtee/tmt#1055]
I have 3 files in the sources file for weldr-client, the source tar.gz, the gpg signature of the source which is .tar.gz.asc, and the gpg public key to use when verifying the source. While trying to add tmt support I discovered that it gets confused by the .tar.gz.asc file:

    The exception was: ("Failed to run command 'tar --auto-compress --extract -f weldr-client-35.5.tar.gz.asc'. Reason: Command returned '2'.", 'tar --auto-compress --extract -f weldr-client-35.5.tar.gz.asc', 2, '', 'tar: This does not look like a tar archive\ntar: Exiting with failure status due to previous errors\n')

File looks like so:

```
SHA512 (weldr-client-35.5.tar.gz) = 81f4a97ce1f1081ea27d0f5e3513ef705586b5b29ff4f9047e067888af4784472c79203397aa8e567414f3e92a1b1fef55412d0e151adaef01f96cc5ec0cceb3
SHA512 (weldr-client-35.5.tar.gz.asc) = 3116b9481bf43eea82029b15bbc972341ef2cc9e73e97bf60490458085cba0f4fe0adb4651688101ffaa2e41f5f5f112a5a6555be3253d3666faf801e3d157cc
SHA512 (gpg-117E8C168EFE3A7F.key) = 9c760460e3227848c99a1178828512056ac04f518f98bdab5ef36d2aa31c4e5dcda3800d7141cfaf7f2acd0f7f35d4b9971083b6f14f8a36231b74041d4ed88d
```

**Comment by lukaszachy** [comment]
Thanks for the report, will fix as part of #1128

**Comment by lukaszachy** [comment]
Looks as easy fix so lets get it merged much sooner.
